hack/swaggify.sh

```diff
@@ -12,4 +12,5 @@ cat \
     | sed 's/workflowtemplate\./io.argoproj.REPLACEME.v1alpha1./' \
     | sed 's/workflow\./io.argoproj.REPLACEME.v1alpha1./' \
     | sed 's/io.argoproj.REPLACEME.v1alpha1./io.argoproj.workflow.v1alpha1./' \
-    | sed 's/k8s.io./io.k8s./'
+    | sed 's/k8s.io./io.k8s./' \
+    | sed 's/v1alpha1\.v1alpha1\./v1alpha1\./g'
```

Comment:
Maybe remove

Reply:
I think the final swagger has some typo errors in it due to this file (not due to your changes, pre-existing).

Reply:
Will look